sdk/tests/client/high_level.rs

```diff
@@ -6,13 +6,12 @@ use iota_sdk::types::block::{
     payload::{transaction::TransactionEssence, Payload},
 };
 
-use crate::client::{common::setup_client_with_node_health_ignored, node_api::setup_transaction_block};
+use crate::client::node_api::setup_transaction_block;
 
 #[ignore]
 #[tokio::test]
 async fn test_find_inputs_from_transaction_id() {
-    let (_block_id, transaction_id) = setup_transaction_block().await;
-    let client = setup_client_with_node_health_ignored().await;
+    let (_block_id, transaction_id, client) = setup_transaction_block().await;
```

Comment:
I'd rather the fn accept a client, than have it create one.

Reply:
changed